client/components/AddTestToQueueWithConfirmation/index.jsx

```diff
@@ -142,31 +183,72 @@ function AddTestToQueueWithConfirmation({
     };
 
     const renderErrorDialog = () => {
```

Comment:
Seems like we need to rename the dialog now since the new message doesn't seem like an error anymore!

Reply:
Good point, done in https://github.com/w3c/aria-at-app/pull/978/commits/bffd2247824ed3d639b6576f8aad11085d539030. Let me know if you'd like the workshop the name further.